### PR TITLE
Limit AWS AZ selection

### LINKED ISSUE #172: AWS regions are too expansive
AWS availability zone lists now include small geographic areas, and Wavelength (5G tower)  AZs. We need to filter the list back to the classic `/#{region}[a-z]/` list before selecting AZs to launch the EKS cluster in.

### PR DESCRIPTION
**Pull request by bear454**
Fixes #172 

As a result, the happy path test needed to be re-recorded, and to do so I needed to make things easier to run.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ This Ruby on Rails-bsed project uses [rvm](http://rvm.io/rvm/basics) to manage a
 
 4.  Create a local config file
     ```
-    cp config/config.yml.example config/config.yml
+    cp config/exampl_econfig.yml config/config.yml
```
Comment (jesusbv) [inline README.md:39]: I know it is the example but 
s/exampl_econfig/example_config

File: engines/rancher_on_eks/app/models/rancher_on_eks/deployment.rb [hunk 2]
```diff
@@ -306,5 +296,24 @@ def deploy
       # in case Rancher create command gets failed status
       raise StandardError.new("Creating #{ApplicationController.helpers.friendly_type(@type)}: status failed") if Rails.configuration.lasso_error.present? && Rails.configuration.lasso_error != "error-cleanup"
     end
+
+    def select_zones(
+        instance_type: @cluster_size.instance_type,
```
Comment (jesusbv) [inline engines/rancher_on_eks/app/models/rancher_on_eks/deployment.rb:301]: bit nitpicking, should that have 2 spaces indentation ?

File: README.md (hunk 1 above)
Comment (rjschwei) [reply README.md:39]: Should be fixed before merge

**Review by rjschwei — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (bear454) [reply README.md]: Fixed in 8c49d46

File: engines/rancher_on_eks/app/models/rancher_on_eks/deployment.rb (hunk 2 above)
Comment (bear454) [reply engines/rancher_on_eks/app/models/rancher_on_eks/deployment.rb:301]: It's intentionally 4, so the closing parenthesis can be indented to the interior of the method. It's a weird case style-wise but I prefer this to the alternative.

**Pull request merged**